src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -894,6 +977,8 @@ export class SearchSource {
     } else {
       body.fields = filteredDocvalueFields;
     }
+    // @ts-ignore
+    body.fields = body.fields.filter((field) => Boolean(field));
```

Comment:
I take it this is filtering out empty values?

Reply:
I just made it work, I had undefined coming from somewhere, and filtered it out. should be solved differently, but I didn't have so much time I was like DataViewLazy, didn't do all the work :)